### PR TITLE
Update pack_metadata.json

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
@DanilVil requested to change the name of the pack

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
1. @JudahSchwartz No need for RN
2. Build is failing as if the pack was not installed correctly, can you check this?

**Comment by JudahSchwartz** [comment]
looks like a random failure 
errElasticsearchUnavailable","status":503.
Validate failed without the RN. Should I delete and we'll force?

**Comment by ShahafBenYakir** [comment]
Don't see a reason why we need a force merge here.

File: Packs/BmcITSM/pack_metadata.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-    "name": "BmcITSM",
-    "description": "BmcITSM integration allows customers to manage service request, incident, change request, task, problem investigation and known error tickets.",
+    "name": "BMC Helix ITSM",
+    "description": "BMC Helix ITSM allows customers to manage service request, incident, change request, task, problem investigation and known error tickets.",
     "support": "xsoar",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.0.1",
```
Comment (JudahSchwartz) [inline Packs/BmcITSM/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.0.0",
```

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**